Makefile

```
@@ -34,7 +37,13 @@ APP_SIGNING_KEYCHAIN := $(if $(wildcard $(KEYCHAIN_FILE)),$(shell realpath $(KEY
 setup: \
 	$(XCPROJECT) \
 	$(PROJECT)/VPNLib/vpn.pb.swift \
-	$(PROJECT)/VPNLib/FileSync/daemon.pb.swift
+	$(PROJECT)/VPNLib/FileSync/daemon.pb.swift \
+	$(addprefix $(PROJECT)/Resources/,$(MUTAGEN_RESOURCES))
+
+# Mutagen resources
+$(addprefix $(PROJECT)/Resources/,$(MUTAGEN_RESOURCES)):
+	gsutil cp "gs://coder-desktop/mutagen/$(MUTAGEN_VERSION)/$(subst Coder ,,$(notdir $@))" "$@"
```

```suggestion
	gsutil cp "gs://coder-desktop/mutagen/$(MUTAGEN_VERSION)/$(notdir $@)" "$@"
```

`$@` will be like `Coder Desktop/Resources/mutagen-agents.tar.gz` so AFAIK the `subst` call doesn't do anything here

---

I think the space breaks it, with this change it tries to copy from `../v0.18.1/Coder mutagen-agents.tar.gz`

---

I guess `notdir` doesn't function properly with paths with a space in them even if they're escaped :(

You could probably use `$(shell basename "$@")` instead, or if you want to stick with the `subst` you should probably make the non-escaped project name be in a new variable like `PROJECT_UNESCAPED`